JSDemos/Demos/DataGrid/RecordPaging/description.md

```diff
@@ -8,5 +8,9 @@ Enables page navigation.
 - **Page size selector**        
 Changes the page size. To display this element, enable the [showPageSizeSelector](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#showPageSizeSelector) property. You can also define the [allowedPageSizes](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#allowedPageSizes) and specify the initial [pageSize](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/paging/#pageSize) in the [paging](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/paging/) object.
 
-- **Page information**      
-Displays the current page number and total record count. To display page information, enable the [showInfo](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#showInfo) property. You can also use the [infoText](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#infoText) property to customize the information text string.
+- **Page information**           
+Displays the current page number and total record count. Available only in the full mode. To display page information, enable the [showInfo](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#showInfo) property. You can also use the [infoText](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#infoText) property to customize the information text string.
+
+Pager has the full and compact display modes. Additionally, the adaptive (default) mode switches between these two values based on the DataGrid's width. Use the **pager**.[displayMode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/#displayMode) property to specify the mode.
+
+In this demo, you can use the SelectBox to switch between the full and compact display modes. This will result in appearance changes of the page navigator and page size selector and will hide the page information in the compact mode. You can also use the checkboxes to show or hide the page size selector and - if in the full display mode - page information and buttons in the page navigator.
```

Comment:
- SelectBox acts as a drop-down menu. Since this is not a SelectBox demo, it is better to refer to this element by its function.
- "This will result in appearance changes of the page navigator and page size selector and will hide the page information in the compact mode." - I describe changes in appearance in the previous paragraph.
- "and - if in the full display mode - page information and buttons in the page navigator." - This needed elaboration.

```suggestion
In this demo, you can use the Display Mode drop-down menu to switch between the full and compact display modes. You can also use the checkboxes to control the visibility of different pager elements. Note that when the pager is in compact mode, you cannot show page information because this mode doesn't support it. You also cannot hide navigation buttons because they are the only means of page navigation.
```